roles/ceph-common/tasks/installs/install_on_redhat.yml

```diff
@@ -44,11 +46,8 @@
     - ceph
     - ceph-mon
   when:
-    (ceph_origin == "distro" or ceph_stable_rh_storage) and
     mon_group_name in group_names and
-    ansible_pkg_mgr == "yum" and
-    ceph_stable and
-    ceph_stable_release not in ceph_stable_releases
+    ansible_pkg_mgr == "yum"
```

Comment:
we can not remove the check for stable releases because older versions don't have `ceph-mon` or `ceph-osd` packages. Please use the following when statement:

```
  when:
    (ceph_origin == "distro" or
    ceph_stable_rh_storage or
    (ceph_stable and ceph_stable_release not in ceph_stable_releases)) and
    mon_group_name in group_names and
    ansible_pkg_mgr == "yum"
```

Validated this on RHEL 7.2 and RHCS ISO installation.